maths/proth_number.py

```diff
@@ -66,12 +58,6 @@ def proth(number: int) -> int:
 
 
 if __name__ == "__main__":
-    for number in range(11):
-        value = 0
-        try:
-            value = proth(number)
-        except ValueError:
-            print(f"ValueError: there is no {number}th Proth number")
-            continue
+    import doctest
 
-        print(f"The {number}th Proth number: {value}")
```

Comment:
Why remove user interaction?!?

Reply:
CONTRIBUTING.md said no print statements, but I'll revert this too

Reply:
Read closer.
1. This is NOT a function or method.
2. This is not part of the algorithm.

Reply:
Right, I got you